src/devices/src/virtio/block/device.rs

```diff
@@ -412,43 +424,16 @@ impl VirtioDevice for Block {
     }
 }
 
-#[cfg(test)]
-pub(crate) mod tests {
-    use std::fs::metadata;
-    use std::os::unix::io::AsRawFd;
-    use std::thread;
-    use std::time::Duration;
-    use std::u32;
-
-    use super::*;
-    use crate::virtio::queue::tests::*;
-    use polly::event_manager::{EventManager, Subscriber};
-    use utils::epoll::{EpollEvent, EventSet};
+pub mod block_utils {
```

Comment:
All of these test utils that need to be used outside of unit-tests (I am guessing in your case they'd be used in fuzzing), should go to a module uniformly called `test_utils` across crates.

In virtio-case these utils will be smth like:
```rust
use device::virtio::queue::test_utils::*;
use device::virtio::block::test_utils::*;
use device::virtio::net::test_utils::*;
// etc.
```

Naming them uniformly makes it easy to find them and to also exclude them from coverage same as we exclude `tests` modules.

Reply:
Ideally they could even live in their own `test_utils.rs` subfiles.